src/main.py

```diff
@@ -128,17 +128,23 @@ def commit_changes(changes):
 
 
 DISCLAIMER = (
-    "<i>(This pull request was automatically generated by "
-    "[flathub/flatpak-external-data-checker]"
+    "🤖 This pull request was automatically generated by "
+    "[flathub/flatpak-external-data-checker] "
     "(https://github.com/flathub/flatpak-external-data-checker). "
-    "Please contact or mention `@barthalion` or `@wjt` if you have any "
-    "questions or complaints.)</i>"
+    "Please [open an issue] "
+    "(https://github.com/flathub/flatpak-external-data-checker/issues/new) "
```

Comment:
I guess it will result in broken markup. Put the hyperlink on single line just in case.

Reply:
I've removed the whitespace but kept them on separate lines, as they were before.